subsys/net/lib/lwm2m/Kconfig

```diff
@@ -123,22 +130,29 @@ config LWM2M_FIRMWARE_UPDATE_PULL_SUPPORT
 	bool "Firmware Update object pull support"
 	default y
 	depends on LWM2M_FIRMWARE_UPDATE_OBJ_SUPPORT
+	select HTTP_PARSER_URL_ONLY
```

Comment:
This is a problem in case the application also wants to use the HTTP library, as it will force Zephyr to use PARSER_URL_ONLY.

Wonder if it should depend on it instead (HTTP_PARSER || HTTP_PARSER_URL_ONLY), but the problem is that someone needs to set the option.